cert/selfsigned.go

```diff
@@ -250,34 +252,50 @@ func GenCertPair(org, certFile, keyFile string, tlsExtraIPs,
 	derBytes, err := x509.CreateCertificate(rand.Reader, &template,
 		&template, &priv.PublicKey, priv)
 	if err != nil {
-		return fmt.Errorf("failed to create certificate: %v", err)
+		return nil, nil, fmt.Errorf("failed to create certificate: %v",
```

Comment:
While we're touching this code, can we please fix the formatting of the multi-line function calls on the line above and a few more below?

Reply:
totally

Reply:
I meant the formatting of
```
	derBytes, err := x509.CreateCertificate(rand.Reader, &template,
		&template, &priv.PublicKey, priv)
```
which should be
```
	derBytes, err := x509.CreateCertificate(
		rand.Reader, &template, &template, &priv.PublicKey, priv,
	)
```
Same below for some `pem.Encode()` calls.